pkg/transfer/local/pull.go

```diff
@@ -255,24 +255,22 @@ func (ts *localTransferService) pull(ctx context.Context, ir transfer.ImageFetch
 }
 
 func fetchHandler(ingester content.Ingester, fetcher remotes.Fetcher, pt *ProgressTracker) images.HandlerFunc {
-	return func(ctx context.Context, desc ocispec.Descriptor) (subdescs []ocispec.Descriptor, err error) {
+	return func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
```

Comment:
It's unclear to me why named returns were used here; after dropping the case scope this becomes an error (due to `err` previously being shadowed).

I've elected to drop this as everything becomes more clear with explicit returns in this otherwise trivial function.